### PR TITLE
Add "homestead share" command (updated)

### PR DESCRIPTION
**Pull request by vicdelfant**
Improved version of an earlier PR, #147. This PR adds a `homestead share` command that uses Vagrant's built-in sharing capability for making sites externally available.

Usage is pretty straight-forward: create an account with Atlas and run `homestead share`. Doesn't require any additional sites to be added to Homestead.yaml (as opposed to the earlier version of this PR) and can be used with both the default dynamic hostnames generated by Atlas as well as custom domains.

### CODE REVIEW

File: scripts/stop-serve.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+if [ -f "/etc/nginx/sites-available/$1" ]; then
+	rm "/etc/nginx/sites-enabled/$1"
+	rm "/etc/nginx/sites-available/$1"
+
+	service nginx restart
+	service php5-fpm restart
+	service hhvm restart
+fi
```
Comment (GrahamCampbell) [inline scripts/stop-serve.sh]: missing newline

Comment (vicdelfant) [inline scripts/stop-serve.sh]: Before the `then` you mean? Added it in 0dedb02

**Comment by taylorotwell** [comment]
I hate to say it but this is just so much to maintain I would rather avoid it.

**Comment by vicdelfant** [comment]
No prob, decision is up to you of course.

**Pull request closed without merging**